Integrations/ThreatQ_v2/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## [Unreleased]
-
+  - Fixed results numbering in ***threatq-get-all-adversaries***, ***threatq-get-all-indicators*** and ***threatq-get-all-events*** commands.
+  - Added unit-testing to the integration.
```

Comment:
this shouldn't be in the change log

Reply:
fixed (can be seen under threatq-integration branch)